src/components/Projects/hooks/useInfiniteProjectsScroll.ts

```diff
@@ -7,10 +7,19 @@ import {
 const useInfiniteProjectsScroll = ( { params: newInputParams, enabled }: object ): object => {
   const baseParams = {
     ...newInputParams,
-    per_page: 50,
+    per_page: 20,
```

Comment:
Does this actually have an effect on performance, like is the time it takes to retrieve one such request smaller?
I would assume that overall if we have to load a large list of projects needing to splice the dataset into smaller chunks (as with 50) is better for performance.

Reply:
You're welcome to experiment with that! I made the `PerformanceDebugView` thinking it would be helpful for future developers to try out theories like this. Generally though, all API calls should reflect what we actually want to show the user -- in this case 20 is already about double the number of project items they'll see on first load on an iPhone device with the current UI, so it seems more than sufficient.